admin/acceptance_tests/__init__.py

```diff
@@ -65,3 +65,9 @@ def check_checkboxes(self, form, name, expected):
             assert exp['label'] == list(label.stripped_strings)[0]
             assert exp['value'] == checkbox['value']
             assert exp['checked'] == field.checked
+
+    def getFirstFieldNamed(self, form, name):
+        return form.fields.get(name)[0]
+
+    def setFirstFieldNamed(self, form, name, value):
+        form.fields.get(name)[0].value__set(value)
```

Comment:
Note that I do not find these functions really useful

Reply:
They ought to be stub for further improvement...
For example getFirstFieldNamed(self, form, name) could become checkFirstFieldNamed((self, form, name, expected) .
I had to replace a few "form[name] = value" by "form.fields.get(name)[0] = value" then by "form.fields.get(name)[0].value__set(value)", so I thought I could be a good thing to be able to do the replacement in one place.